### ISSUE TITLE
BUG: DB not auto selected after creation in sidebar

### ISSUE THREAD

**Issue by mcanepa** [beekeeper-studio/beekeeper-studio#1954]
I've created a database via "Add Database" plus icon. After the dialog closes, I see the new database on the dropdown as selected. Then I opened a new tab and run a crate table statement. I get an error that "no database is selected". I had to manually select another database and then select he newly created one ir order to run the sql

**Comment by mcanepa** [comment]
I've just created a database via SQL and the same behavior occurs.

```
CREATE DATABASE `test_database`;
USE `test_database`;
```

I expected the left panel to update and show `test_database` to be selected.

**Comment by not-night-but** [comment]
@mcanepa I'm not able to reproduce this. What database are you using? What OS? What version of BKS? If this is happening, then it's a bug. Although running the SQL manually will not auto select the DB in the sidebar.

**Comment by mcanepa** [comment]
I'm using Beekeeper 5.0.9 with MariaDB 10.11 on a Debian 12

**Comment by not-night-but** [comment]
@mcanepa I still can't reproduce this on any database. Are there any errors in the console? Try checking the log files in `~/.config/beekeeper-studio/logs/`

**Comment by mcanepa** [comment]
@not-night-but You are right. I confirm is not happening on v5.0.9 altough it was happening when I posted the issue.

So that part is solved

But I believe that when I run the `USE` statement, the panel should reflect my selection.

**Comment by not-night-but** [comment]
@mcanepa I see the USE statement as temporarily setting the database for the current query that is being run. Having it switch the whole state of the application could cause further issues and confusion for other users.
